.gitignore

```diff
@@ -1,8 +1,8 @@
 *.db
 *.env
 *.mo
-fixtures/*.json
-!fixtures/??_*.json
+benefits/core/migrations/0002_*.py
+!benefits/core/migrations/0002_sample_data.py
```

Comment:
I like this approach. Gives us the ability to have any number of `0002_migration.py` files locally!